ivw_integration.install

```diff
@@ -27,3 +27,87 @@ function ivw_integration_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Add missing field columns to field tables.
+ *
+ * @param string $field_type
+ *   The field types id.
+ *
+ * @return array
+ *   Gathered messages.
+ */
+function ivw_integration_add_missing_field_columns($field_type) {
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $database_schema = \Drupal::database()->schema();
+
+  // Get all of the field storage entities of type ivw_integration_settings.
+  $field_storage_configs = $entity_type_manager
+    ->getStorage('field_storage_config')
+    ->loadByProperties(['type' => $field_type]);
+
+  foreach ($field_storage_configs as $key => $field_storage) {
+    // The name of the field.
+    $field_name = $field_storage->getName();
+    $field_schema = $field_storage->getSchema();
+
+    // The entity type to which this field is attached.
+    $target_entity_type_id = $field_storage->getTargetEntityTypeId();
+    $target_entity_type = $entity_type_manager->getDefinition($target_entity_type_id);
+    $target_entity_type_storage = $entity_type_manager->getStorage($target_entity_type_id);
+
+    // Table mapping is used to generate the table names of the found fields.
+    $table_mapping = $target_entity_type_storage->getTableMapping();
+
+    // Get the table name, that needs to be updated.
+    $field_table_names = [$table_mapping->getDedicatedDataTableName($field_storage)];
```

Comment:
No, this method returns all field tables of the given target_entity, not just the ones for a specific field type